<launch>
	<node pkg="vehicle_detector" type="vehicle_detection" name="vehicle_detector" output="screen">
	<!--node pkg="vehicle_detector" type="vehicle_detection_blackcar" name="vehicle_detector" output="screen"-->
		<param name="image_transport" value="compressed" />
		<param name="car_model" value="$(find vehicle_detector)/model/car.xml"/>
		<param name="bus_model" value="$(find vehicle_detector)/model/bus.xml"/>
	</node>

	<!--node pkg="vision_sandbox" type="ground_extraction" name="ground_extraction" output="screen">
		<param name="image_transport" value="compressed" />
	</node-->
</launch>
